SQLQuery_ZooManagement.sql

```diff
@@ -0,0 +1,164 @@
+CREATE Database ZooManagement;
+
+Use ZooManagement;
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species VARCHAR(150),
+SpeciesNumber INT,
+AnimalsNumber INT,
+);
+
+CREATE TABLE Visitors
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Day VARCHAR(150),
+NumberOfVisitors INT,
+);
+
+CREATE TABLE Tickets
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Price INT,
+Validity VARCHAR(150),
+);
+
+CREATE TABLE Caretakers
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR(150),
+);
+
+CREATE TABLE ZooMentenance
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+AreaOfWork VARCHAR(150),
+);
+
+CREATE TABLE WorkDistribution

```

Comment:
The consistency in following he same naming convention is important. There are a few table names as plural (Caretakers, Tickets) and some singular (ZooMentenance, WorkDistribution). It's a best practice to name them all plural as inside a table we store multiple records of that type.